x-pack/plugins/apm/public/components/app/service_inventory/service_list/index.tsx

```diff
@@ -63,19 +65,51 @@ export function getServiceColumns({
   comparisonData,
   breakpoints,
   showHealthStatusColumn,
+  showAlertsColumn,
+  link,
 }: {
   query: TypeOf<ApmRoutes, '/services'>['query'];
   showTransactionTypeColumn: boolean;
   showHealthStatusColumn: boolean;
+  showAlertsColumn: boolean;
   comparisonDataLoading: boolean;
   breakpoints: Breakpoints;
   comparisonData?: ServicesDetailedStatisticsAPIResponse;
+  link: any;
 }): Array<ITableColumn<ServiceListItem>> {
   const { isSmall, isLarge, isXl } = breakpoints;
   const showWhenSmallOrGreaterThanLarge = isSmall || !isLarge;
   const showWhenSmallOrGreaterThanXL = isSmall || !isXl;
 
   return [
+    ...(showAlertsColumn
+      ? [
+          {
+            field: ServiceInventoryFieldName.AlertsCount,
+            name: '',
+            width: `${unit * 4}px`,
+            sortable: true,
+            render: (_, { serviceName, alertsCount }) => {
+              if (!alertsCount) {
+                return null;
+              }
+
+              return (
+                <EuiBadge
+                  iconType="alert"
+                  color="danger"
+                  href={link('/services/{serviceName}/alerts?', {
```

Comment:
nit: I believe  `link` should handle the `?`

Reply:
totally right, thanks for spotting this